src/raftstore/store/fsm/apply.rs

```diff
@@ -1003,11 +1004,10 @@ impl ApplyDelegate {
         let cmd_type = request.get_cmd_type();
         if cmd_type != AdminCmdType::CompactLog && cmd_type != AdminCmdType::CommitMerge {
             info!(
-                "{} execute admin command {:?} at [term: {}, index: {}]",
-                self.tag,
-                request,
-                ctx.exec_ctx.as_ref().unwrap().term,
-                ctx.exec_ctx.as_ref().unwrap().index,
+                "execute admin command {:?}", request;
```

Comment:
Why use two style?

Reply:
Because `request` can't be formatted in `"request" => request`.